### DISCUSSION TITLE
Floating Icon Size / ClipArt Dock Selection

### DISCUSSION THREAD

**Discussion by dchester8** [pubpub-zz/ppInk#252] [General]
Hi,
Just recently found the program and its an incredible presentation tool.  I had a few questions about some items.

1. Can the floating window icon size be adjusted?

2. Can the floating window icon 's transparency and/or logo be changed?

3. The ClipArt option in the settings toggles not just the clipart but the arrows, shapes and numbered circles.  Is there a way in any of the ini's to choose what I want on the dock and not in the dock?  I'm trying to reduce the size of the dock but still keep some of the clipart options like the circled numbers and custom icons but remove the shapes, arrows and lines.

Again thank you for the program its been helping me tremendously!

**Comment by pubpub-zz** [comment]
Happy you are enjoying. Do not hesitate to star it 😊

the floating window width/heigth/transparency can be modified directly in  config.ini:
```
# Calll Window position: Top,Left,Width(=Height),transparency
Window_POS = 990,1871,48,50
```
the logo of the calling windows can be modified modifying the image FloatingCall in ppink folder (PNG,JPG,BMP extensions can be used)

For last point, sorry, I have not implemented a show/hide setup per tool (quite long job). 
remember that even if the tools are not displayed the hotkeys are still active.
in the latest beta version ( https://github.com/pubpub-zz/ppInk/files/11105092/ppInk.SaveIni.zip ), you have 2 "pages" of pens : maybe you wll be able to divide by 2 the number of visible pens at once (see #240 )
